configuration/editors.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
+
+OpenHAB has a great development environment with the "openHAB Designer".
+But in some cases you want to use another editor to make changes to the configuration files of openHAB.
+To make this more effectiv there are some files to enable syntax highlighting for openHAB-files in these editors.
+
+{::options toc_levels="2"/}
+
+* TOC
+{:toc}
+
+## mcedit
+
+mcedit is an editor which comes with mc (Midnight Commander).
+
+
+### Installing the syntax-files
+
+- download copy the syntax-files to */usr/share/mc/syntax/*
+- https://code.google.com/p/openhab-samples/source/browse/syntaxhl/mc/?repo=wiki
+
+- insert the following lines to the file *Syntax* in */usr/share/mc/syntax/*
+```
+    file ..\*\\.(items)$ openHAB\sItems 
+    include openhab-items.syntax  
+     
+    file ..\*\\.(sitemap)$ openHAB\sSitemap 
+    include openhab-sitemap.syntax
+     
+    file ..\*\\.(persist)$ openHAB\sPersistence
+    include openhab-persist.syntax
+     
+    file ..\*\\.(rules)$ openHAB\sRules
+    include openhab-rules.syntax 
+```
+- edit the Debian-line from
+`file (rules|rocks)$ Debian\srules`
+to 
+`file (rocks)$ Debian\srules`
+because it interferes with openHABs rules-files.
+
+### Screenshots
+![Items](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_items.png "Items")
+![Rules](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_rules.png "Rules")
+![Sitemap](http://wiki.openhab-samples.googlecode.com/hg/screenshots/syntaxhl_mc_sitemap.png "Sitemap")
+
+## Notepad++
+
+Notepad++ Version 6.2 or above is required to support UDL2 (User Defined Language v2).
```

Comment:
I found the GitHub repository copy from the old Google Code: https://github.com/thefrip/openhab-samples